update-server/Makefile

```
@@ -23,7 +23,7 @@ br_ssh_opts ?= $(default_ssh_opts)
 
 .PHONY: setup
 setup:
-	$(pipenv_envvars) pipenv sync $(pipenv_opts)
+	$(pipenv_envvars) $(pipenv) sync $(pipenv_opts)
```

Nice catch!

I think this can be reduced to:

```suggestion
	$(pipenv) sync $(pipenv_opts)
```

Because `$(pipenv)` is defined like:

https://github.com/Opentrons/opentrons/blob/5da32381c0cc94d622d4e3e064d10d80aaf0187d/scripts/python.mk#L8